samigo/samigo-impl/src/java/org/sakaiproject/samigo/impl/SamigoETSProviderImpl.java

```diff
@@ -0,0 +1,441 @@
+/**********************************************************************************
```

Comment:
The License header here should be the one from ECLv2-header-template.txt in the root of the project.

This is the older subversion header.